client-java-contrib/admissionreview/pom.xml

```diff
@@ -13,33 +13,22 @@
     </parent>
     <version>12.0.1-SNAPSHOT</version>
 
-    <dependencies>
```

Comment:
I think you need to retain these dependencies. The compile is failing in CI/CD.

Reply:
fixed the compiling failure